sfdx-project.schema.json

```diff
@@ -199,6 +203,11 @@
         }
       }
     },
+    "packageDirectory.functions": {
+      "title": "Functions To Package",
+      "type": "array",
+      "description": "Relative path to functions to be packaged."
```

Comment:
should this be ...`to functions directories to be`... ? or some verbiage to clarify there should be a relative path to each function directory to be packaged?

Reply:
Agree, changing.  Thank you.